core/sections/clause_9_security-considerations.adoc

```diff
@@ -68,15 +68,15 @@ Most of the operations defined in this standard are use the safe HTTP GET operat
 
 |===
 
-The resources and contained information in more detail:
+The following is more detailed information on the resources:
 
-* The landing page contains links to the API endpoints and so leads to all other resources the API offers.
+* The landing page can include links to endpoints, that are not defined in the API - Processes Standard, that can lead to all the other resources that the containing API might offer.
 
 * The list of conformance classes could contain information about extensions like "dismiss" that pose additional security issues.
 
 * The process list contains process identifiers and links to the respective process descriptions.
 
-* The process description contains all necessary information needed to execute a process. This information can be used to send an JSON execute request to the API that will pass initial sanity checks, for example checks for the correct input/output identifiers. If this barrier is taken by an attacker, issues as discussed in section <<sec_cons_execute>> can occur.
+* The process description contains all necessary information needed to execute a process. This information can be used to send a JSON execute request to the Process API that will pass initial sanity checks, for example checks for the correct input/output identifiers.  The ability to pass values to the server is vulnerable to injection attacks and issues as discussed in section <<sec_cons_execute>> can occur.
```

Comment:
"Processes API" instead of "Process API" to align with other uses elsewhere

Reply:
Fixed.